Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
```

Comment:
should these changes be in `vnext\Directory.Build.props` so that customer projects have this as well?

Reply:
If by "customer projects" you mean the ones generated from templates, it is taken care of.
Placing these properties at the root PROPS allows for them to be ingested by projects in the `packages` hierarchy.